build/tools/Automation/Python/Constants.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
```

Comment:
nit: all of the new files add should have a license header

Reply:
Updated.